spaceship.zsh

```diff
@@ -76,13 +76,27 @@ SPACESHIP_DIR_COLOR="${SPACESHIP_DIR_COLOR:="cyan"}"
 SPACESHIP_GIT_SHOW="${SPACESHIP_GIT_SHOW:=true}"
 SPACESHIP_GIT_PREFIX="${SPACESHIP_GIT_PREFIX:="on "}"
 SPACESHIP_GIT_SUFFIX="${SPACESHIP_GIT_SUFFIX:=""}"
-SPACESHIP_GIT_UNCOMMITTED="${SPACESHIP_GIT_UNCOMMITTED:="+"}"
-SPACESHIP_GIT_UNSTAGED="${SPACESHIP_GIT_UNSTAGED:="!"}"
-SPACESHIP_GIT_UNTRACKED="${SPACESHIP_GIT_UNTRACKED:="?"}"
-SPACESHIP_GIT_STASHED="${SPACESHIP_GIT_STASHED:="$"}"
-SPACESHIP_GIT_UNPULLED="${SPACESHIP_GIT_UNPULLED:="⇣"}"
-SPACESHIP_GIT_UNPUSHED="${SPACESHIP_GIT_UNPUSHED:="⇡"}"
-SPACESHIP_GIT_COLOR="${SPACESHIP_GIT_COLOR:="magenta"}"
+SPACESHIP_GIT_SYMBOL="${SPACESHIP_GIT_SYMBOL:=" "}"
```

Comment:
It looks great and many themes are using it. But don't go well with normal fonts. This is how it looks with some non-powerline fonts

#### Inconsolata Regular 12
![Inconsolata Regular 12](http://i.imgur.com/yk74RN6.png)

#### DejaVu Sans Mono Book 12
![DejaVu Sans Mono Book 12](http://i.imgur.com/Dpvh5jj.png)